core_api/src/routes/chat.py

```diff
@@ -112,7 +112,7 @@ def simple_chat(chat_request: ChatRequest) -> ChatResponse:
     return ChatResponse(response_message=ChatMessage(text=response.text, role="ai"))
 
 
-@chat_app.post("/rag", tags=["chat"], response_model=ChatResponse)
+@chat_app.post("/rag", tags=["chat"])
```

Comment:
Does the response model need to be returned?

Reply:
not if it is in the annotation (below). ofc its not doing any harm either, happy to out it back?